### ISSUE TITLE
Tab Order In All Forms

### ISSUE THREAD

**Issue by sdaigle2** [sdaigle2/pdi-database#16]
We need to work on the Tab Order in all of the forms.  If a control is not listed in this list, it should have a TabStop property to **No** so that it does not appear in the tab order

## frmHoursLogOverhead
**Tab Order**
1. cmbEmployee
2. cmbCustomer
3. cmbJob
4. cmbPhase
5. txtTimeSpent
6. txtDescription
7. cmdSave

## frmJobs
**Tab Order**
1. cmbCustID
2. txtJobName
3. txtJobDescription
4. txtJobStatus
5. cmdSave

## frmGenReports
**Tab Order**
1. cboPeriod
2. txtStartDate
3. txtEndDate
4. btnSageHoursReport
5. btnHoursByEmployee
6. btnHoursByEmployeeByJob
7. btnHoursbyJob

## frmCreateTask
**Tab Order**
1. cboCustomer
2. cboJobID
3. cboPhase
4. txtTaskTitle
5. txtDescription
6. cboAssignedTo
7. txtHoursToCompletion
8. cboTimeRange
9. txtActualHours _(if it is visible, sometimes this control is hidden)_
10. chkComplete _(if it is visible, sometimes this control is hidden)_
11. txtCompleteDate _(if it is visible, sometimes this control is hidden)_
12. btnCreatTask

## Other Forms
frmCustomers - no changes needed
frmEmployees - no changes needed

**Comment by smsu07** [comment]
Please check

**Comment by sdaigle2** [comment]
Everything looks good.  I made a minor change to frmJobs and commited/pushed to github.
> in frmHoursLogOverhead, cmbWeekday should be after cmbEmployee and before cmbCustomer

If this commit conflicts with anything you did, please revert and make the change manually.
